packages/helpers/src/strings/stripTagsFromHtmlString.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Strips tags from HTML nodes.
+ *
+ * @param {NodeListOf<ChildNode>} nodes The nodes.
+ * @param {string[]} [allowedTags] Optional. The allowed tags.
+ *
+ * @returns {void}
+ */
+const stripHtmlTagsFromNodes = ( nodes, allowedTags = [] ) => {
+	// Keep a list, otherwise the loop can go wrong when a node gets replaced.
+	const replaceList = [];
+
+	nodes.forEach( node => {
+		if ( node.nodeType !== Node.ELEMENT_NODE ) {
+			return;
+		}
+
+		const tag = node.nodeName.toLowerCase();
```

Comment:
We are not checking for the JS event listeners on the HTML tag such as `onerror` and `onclick`.